src/components/Navigation.js

```diff
@@ -31,9 +32,9 @@ const Navigation = () => {
           <NavItem>
```

Comment:
We'll want to make it so this only renders if you're logged in, we don't want a profile link to show up if `username` is empty.

Reply:
The link for the User Page is already set within the ternary operator to not appear unless a user is logged in.